src/sentry/demo/demo_org_manager.py

```diff
@@ -58,8 +58,23 @@ def create_demo_org(quick=False) -> Organization:
             )
             react_project.add_team(team)
 
-            populate_org_members(org, team)
+            if settings.DEMO_MOBILE_PROJECTS:
```

Comment:
@roggenkemper you should add a test for this in `test_demo_org_manager.py`

Reply:
added a test to check if all projects are created!